src/platforms/common/configuration/options.mdx

```diff
@@ -312,14 +312,22 @@ Configures a separate proxy for outgoing HTTPS requests. This value might not be
 
 </ConfigKey>
 
-<ConfigKey name="cache-directory-path" supported={["dotnet", "android"]}>
+<ConfigKey name="cache-directory-path" supported={["dotnet", "android"]} notSupported={["unity"]}>
 
 Specifies a local directory used for cacheing payloads. When this option is enabled (that is, when the directory is set), the Sentry SDK will persist envelopes locally before sending to Sentry. This configuration option is particularly useful if you expect your application to run in environments where internet connectivity is limited.
 
 Default: not set (caching is disabled).
 
 </ConfigKey>
 
+<ConfigKey name="cache-directory-path" supported={["unity"]}>
+
+Specifies a local directory used for cacheing payloads. When this option is enabled (that is, when the directory is set), the Sentry SDK will persist envelopes locally before sending to Sentry. This configuration option is particularly useful if you expect your application to run in environments where internet connectivity is limited.
+
+Default: Set to `Application.persistentDataPath` (caching is enabled).
```

Comment:
```suggestion
Default: Varies per SDK, for .NET and Android, the default is not set (caching is disabled). For Unity, the default is set to `Application.persistentDataPath` (caching is enabled).
```

Reply:
All Mobile SDKs have offline caching enabled by default.
Java and .NET have it disabled by default.